avocado/core/loader.py

```diff
@@ -694,8 +693,7 @@ def ignore_broken(klass, uid, params=None):
                                                 subtests_filter)
             else:
                 if os.access(test_path, os.X_OK):
-                    return self._make_test(test.SimpleTest,
-                                           pipes.quote(test_path))
+                    return self._make_test(test.SimpleTest, test_path)
```

Comment:
@ldoktor, could you clarify why this:
`avocado run "foo bar" => executes SimpleTest "foo bar" (path 1)`
is a requirement?

In my understanding, if you run such test, `foo` is the filename and `bar` is it's argument. Unless you escape the whitespace.